db/migrate/20160712160707_create_games.rb

```diff
@@ -0,0 +1,16 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.string :name
+      t.string :state
+ 
+      t.integer :white_player_id
+      t.integer :black_player_id
+      t.integer :winning_player
+
+
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
Can you explain some of the decisions made here vs. the model spreadsheet:

https://docs.google.com/spreadsheets/d/1hijYjebn6_fSKAK5FeJLfVqfIGHHE51-iRgSyJNuEDQ/edit#gid=0